conf/monitorix.conf

```
@@ -385,17 +385,7 @@ secure_log_date_format = %b %e
         mta = postfix
         greylist = milter-greylist
         rigid = 0, 0, 0, 0, 0
-        limit = 1, 1000, 1000, 1000, 1000
-        <alerts>
-                delvd_enabled = n
-                delvd_timeintvl = 60
-                delvd_threshold = 100
-                delvd_script = /path/to/script.sh
-                mqueued_enabled = n
-                mqueued_timeintvl = 3600
-                mqueued_threshold = 100
-                mqueued_script = /path/to/script.sh
-        </alerts>
+        limit = 1, 1000, 1000, 1000, 1000       
```

Maybe would be good to configure this as it was in the README.

```suggestion
        limit = 1, 1000, 1000, 1000, 1000
        <alerts>
                delvd_enabled = y
                delvd_timeintvl = 60
                delvd_threshold = 100
                delvd_script = /etc/monitorix/monitorix_alerts_scripts/mail_delvd.sh
                mqueued_enabled = y
                mqueued_timeintvl = 3600
                mqueued_threshold = 100
                mqueued_script = /etc/monitorix/monitorix_alerts_scripts/mail_mqueued.sh
        </alerts>
```

---

Alerts can be used in every monitored parameter (load, CPU, filesystem, etc) - but it is hard to define the alert limits because it depends on your system's goal. For example, if you run a newsletter on your server the high queue size is common - or if you run CI jobs the high load is also normal too.